angularjs-portal-home/src/main/webapp/my-app/search/partials/search-results.html

```diff
@@ -1,144 +1,250 @@
-      No MyUW results. <a href="apps">Try browsing instead?</a>
-    </div>
-    <div ng-repeat="portlet in myuwFilteredResults = (myuwResults | filter:searchTermFilter | showApplicable:showAll | orderBy:sortParameter | limitTo:searchResultLimit)" class="result">
-      <h4><a ng-href="{{getLaunchURL(portlet)}}" target="{{::portlet.target}}">{{ portlet.title }}</a> <small ng-if='GuestMode && !portlet.canAdd'>(login to use)</small></h4>
-      <p>{{ portlet.description }}</p>
-      <p>
-        <a ng-click="addToHome(portlet)"
-           ng-if="portlet.canAdd && !portlet.hasInLayout && !GuestMode"
-           class="add">
-          <i class="fa fa-plus"></i> Add to home
-        </a>
-        <span ng-if="portlet.canAdd && portlet.hasInLayout && !GuestMode"
-              class="added">
-          <i class="fa fa-check"></i> Added to home
-        </span>
-        <button class='btn btn-link' style='padding:0;' title='See more about {{portlet.title}}' ng-click='navToDetails(portlet, "Search")'>Details</button>
-        <span><rating ng-model="portlet.rating" readonly="true" class="rating"></rating></span>
-      </p>
-    </div>
-    <div class="marketplace-load-more" hide-while-loading ng-show="myuwFilteredResults.length > searchResultLimit">
-      <button class="btn btn-primary btn-lg"
-              ng-click="searchResultLimit = searchResultLimit + 20;"
-              hide-while-loading
-              ng-show="myuwFilteredResults.length > searchResultLimit"
-             >Load more MyUW results</button>
-    </div>
-  </div>
+    <!-- ALL RESULTS -->
+    <md-tab ng-click="filterTo('all')">
+      <md-tab-label>
+        All&nbsp;&nbsp;<span class="badge">{{ totalCount }}</span>
+      </md-tab-label>
+      <md-tab-body>
+        <md-content layout-padding>
+          <!-- MyUW results -->
+          <div class='search-results-container'>
+            <h4 class="header">MyUW</h4>
+            <hr>
+            <loading-gif data-object='myuwResults'></loading-gif>
+            <div ng-show="myuwResults.length != 0 && myuwFilteredResults.length == 0" class='no-result'>
+              No MyUW results. <a href="apps">Try browsing instead?</a>
+            </div>
+            <div ng-repeat="portlet in myuwFilteredResults = (myuwResults | filter:searchTermFilter | showApplicable:showAll | orderBy:sortParameter | limitTo:searchResultLimit)"
+                 class="result">
+              <h4><a ng-href="{{getLaunchURL(portlet)}}" target="{{::portlet.target}}">{{ portlet.title }}</a> <small ng-if='GuestMode && !portlet.canAdd'>(login to use)</small></h4>
+              <p>{{ portlet.description }}</p>
+              <p>
+                <md-button ng-click="addToHome(portlet)"
+                   ng-if="portlet.canAdd && !portlet.hasInLayout && !GuestMode"
+                   class="md-primary add">
+                  <i class="fa fa-plus"></i> Add to home
```

Comment:
Consider an `aria` tag to help users who don't visually connect the "Add to home" to the context of which portlet will be added. (As in, "Add {{ portlet.title }} to home").